askama_shared/src/filters/markdown.rs

```diff
@@ -0,0 +1,17 @@
+use std::borrow::Borrow;
```

Comment:
I'd prefer to put this in an inline module in `filters/mod.rs`, it's a bit too short to merit its own file.